### PR TITLE
Getting started documentation provides more context.

### PR DESCRIPTION
**Pull request by brianquinlan**

### CODE REVIEW

File: GETTING-STARTED.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Getting Started
 
+**Django Deploy** is a tool that that allows you to deploy 
+[Django](https://www.djangoproject.com/) applications on
+[Google Cloud Platform](https://cloud.google.com/).
+
+This guide will step you through:
+1. installing the prerequisites for **Django Deploy**
```
Comment (enriquejosepadilla) [inline GETTING-STARTED.md]: nit: Make the first word capitalized.

File: GETTING-STARTED.md [hunk 2]
```diff
@@ -31,8 +41,13 @@ $ pip install django-cloud-deploy
 
 ## Creating a New Project
 
+At the end of this step, you will have a skeleton Django project that is
+deployed on
+[Kubernetes Engine](https://cloud.google.com/kubernetes-engine/). You can
```
Comment (enriquejosepadilla) [inline GETTING-STARTED.md]: The last sentence seems unnecessary.

File: GETTING-STARTED.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
 # Getting Started
 
+**Django Deploy** is a tool that that allows you to deploy
```
Comment (wangxf123456) [inline GETTING-STARTED.md]: Double "that"

File: GETTING-STARTED.md (hunk 2 above)
Comment (brianquinlan) [reply GETTING-STARTED.md]: I like it. I changed the wording a bit but done.

**Pull request merged**